<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dcTermClass</class>
 <widget class="QMainWindow" name="dcTermClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dcTerm</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuBit_Rate">
     <property name="title">
      <string>Bit Rate</string>
     </property>
     <addaction name="action1200"/>
     <addaction name="action2400"/>
     <addaction name="action4800"/>
     <addaction name="action9600"/>
     <addaction name="action19200"/>
     <addaction name="action38400"/>
     <addaction name="action57600"/>
     <addaction name="action115200"/>
    </widget>
    <widget class="QMenu" name="menuData_Bits">
     <property name="title">
      <string>Data Bits</string>
     </property>
     <addaction name="action5"/>
     <addaction name="action6"/>
     <addaction name="action7"/>
     <addaction name="action8"/>
    </widget>
    <widget class="QMenu" name="menuStop_Bits">
     <property name="title">
      <string>Stop Bits</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
    </widget>
    <widget class="QMenu" name="menuParity">
     <property name="title">
      <string>Parity</string>
     </property>
     <addaction name="actionEven"/>
     <addaction name="actionOdd"/>
     <addaction name="actionNone"/>
    </widget>
    <widget class="QMenu" name="menuFlow_Control">
     <property name="title">
      <string>Flow Control</string>
     </property>
     <addaction name="actionNo_Flow_Control"/>
     <addaction name="actionHardware_Control"/>
     <addaction name="actionSoftware_Control"/>
    </widget>
    <addaction name="menuBit_Rate"/>
    <addaction name="menuData_Bits"/>
    <addaction name="menuParity"/>
    <addaction name="menuStop_Bits"/>
    <addaction name="menuFlow_Control"/>
   </widget>
   <widget class="QMenu" name="menuPort">
    <property name="title">
     <string>Port</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuPort"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionParityy">
   <property name="text">
    <string>Parityy</string>
   </property>
  </action>
  <action name="action1200">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1200</string>
   </property>
  </action>
  <action name="action2400">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2400</string>
   </property>
  </action>
  <action name="action4800">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>4800</string>
   </property>
  </action>
  <action name="action9600">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action19200">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>19200</string>
   </property>
  </action>
  <action name="action38400">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>38400</string>
   </property>
  </action>
  <action name="action57600">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>57600</string>
   </property>
  </action>
  <action name="action115200">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>115200</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="actionEven">
   <property name="text">
    <string>Even</string>
   </property>
  </action>
  <action name="actionOdd">
   <property name="text">
    <string>Odd</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionCurrent_Settings">
   <property name="text">
    <string>Current Settings</string>
   </property>
  </action>
  <action name="actionNo_Flow_Control">
   <property name="text">
    <string>No Flow Control</string>
   </property>
  </action>
  <action name="actionHardware_Control">
   <property name="text">
    <string>Hardware Control</string>
   </property>
  </action>
  <action name="actionSoftware_Control">
   <property name="text">
    <string>Software Control</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dcTerm.qrc"/>
 </resources>
 <connections/>
</ui>
